ncm-cron/src/main/pan/components/cron/schema.pan

```diff
@@ -38,12 +38,63 @@ type structure_cron_log = {
     'mode' ? string
 } with structure_cron_log_valid(SELF);
 
+@documentation{
+    Validate contents of cron fields
```

Comment:
can you add a (not too trivial) example of what retuns true and false. i read the code twice, i'm serioulsy confused with the foreach loops